aws_marketplace/using_algorithms/automl/AutoML_-_Train_multiple_models_in_parallel.ipynb

```diff
@@ -0,0 +1,635 @@
+    "1. XGBoost GBM (Gradient Boosting Machine)\n",
+    "2. GLM \n",
+    "3. default Random Forest (DRF)\n",
+    "4. Extremely Randomized Forest (XRT)\n",
+    "5. Deep Neural Nets\n",
+    "\n",
+    "Once these models have been trained, it also creates two stacked ensemble models:\n",
+    "1. An ensemble model created using all the models.\n",
+    "2. Best of family ensemble model created using models that performed best in each class/family.\n",
+    "\n",
+    "For more information on how H2O's AutoML works, see [FAQ section of H2O's documentation.](http://docs.h2o.ai/h2o/latest-stable/h2o-docs/automl.html#faq)\n",
+    "\n",
+    "\n",
+    "#### Compatibility\n",
+    "This notebook is compatible only with [H2O-3 Automl Algorithm](https://aws.amazon.com/marketplace/pp/prodview-vbm2cls5zcnky) from AWS Marketplace and an AWS Marketplace subscription is required to successfully run this notebook. \n",
+    "\n",
+    "\n",
+    "#### Usage instructions\n",
+    "You can run this notebook one cell at a time (By using Shift+Enter for running a cell).\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#Let us install necessary H2O library which you would use to load and inspect the model summary.\n",
+    "import sys\n",
+    "!{sys.executable} -m pip install http://h2o-release.s3.amazonaws.com/h2o/rel-wright/10/Python/h2o-3.20.0.10-py2.py3-none-any.whl"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#Import necessary libraries.\n",
+    "import boto3\n",
+    "import re\n",
+    "import os\n",
+    "import errno\n",
+    "import base64\n",
+    "import time\n",
+    "import numpy as np\n",
+    "import pandas as pd\n",
+    "import urllib\n",
+    "from sagemaker import get_execution_role\n",
+    "import json\n",
+    "import uuid\n",
+    "import sagemaker\n",
+    "from time import gmtime, strftime\n",
+    "import urllib.request\n",
+    "from sagemaker import AlgorithmEstimator\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Step 1: Subscribe to AutoML algorithm from AWS Marketplace\n",
+    "\n",
+    "1. Open [H2O-3 Automl Algorithm listing from AWS Marketplace](https://aws.amazon.com/marketplace/pp/prodview-vbm2cls5zcnky?qid=1557245796960&sr=0-1&ref_=srh_res_product_title)\n",
+    "2. Read the **Highlights** section and then **product overview** section of the listing.\n",
+    "3. View **usage information** and then **additional resources**.\n",
+    "4. Note the supported instance types and specify the same in the following cell.\n",
+    "5. Next, click on **Continue to subscribe**.\n",
+    "6. Review **End user license agreement**, **support terms**, as well as **pricing information**.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
```

Comment:
Why is this code cell in the middle of what looks like a step-wise instruction on how to subscribe and prepare?

Reply:
fair enough - moving this code-block post subscription process walkthrough.